### DISCUSSION TITLE
VanUi - VanWindow ?

### DISCUSSION THREAD

**Discussion by Duffscs** [vanjs-org/van#132] [Ideas]
VanUi is an excellent component library.

The modal component is really nice. A "window" component would be an excellent addition to library.

The floating window could be resizable, closable and draggable, like this.

![window](https://github.com/vanjs-org/van/assets/59032377/63bc1b7d-93d3-4a74-8a5a-a411561da05f)

**Comment by Tao-VanJS** [comment]
This is great! Do you want to submit a PR for it if you already have something implemented?

**Reply by Duffscs** [reply]
It's not implemented yet, it's just a mock-up.
I can give try, although I'm not very comfortable with van at the moment, I don't know if what I'm going to produce will correspond to how the librairy is implemented.

**Reply by Tao-VanJS** [reply]
Gotcha! 3rd party contribution is always welcome 🤗 Don't worry too much about styling. It can always be fixed to make it consistent.

**Reply by Duffscs** [reply]
I've tried something, I'll let you have a look when you have time
#133
